README.md

```diff
@@ -1,3 +1,37 @@
 - A web application on Python.
 - A simple web-based application for Human resource management. 
 - The application with only two entities: “Department” and “Employee”, related as one to many.
+
+**TO DO**
+**APPOINTMENT OF ENVIRONMENTAL VARIABLES** 
+- Open your .bashrc file on your local machine;
+- Define there next environmental variables in separate lines like key="value" pairs:  
+  - `export SECRET_KEY="the_value_of_your_secret_key"`
+  - `export GROW_DB_NAME="the_name_of_your_database"`
+  - `export DB_USER="the_name_of_database's_owner"`
+  - `export DB_PASSWORD="the_database's_password"`
+- Open your settings.py file;
+- Replace in it:
```

Comment:
As I understand this section can be deleted as you have provided all necessary changes in settings file?